1.8/usage/marathon/application-basics.md

```diff
@@ -0,0 +1,143 @@
+---
+post_title: Application Definitions
+nav_title: Application Definitions
+menu_order: 0
+---
+
+Applications are an integral concept in Marathon. Each application typically represents a long-running service, of which there would be many instances running on multiple hosts. An application instance is called a *task*. The *application definition* describes everything needed to start and maintain the tasks. A Marathon application definition creates a DC/OS _service_.
+
+# Hello Marathon: An Inline Shell Script
+
+### Prerequisites
+- [A DC/OS cluster](/docs/1.8/administration/installing/)
+- [The DC/OS CLI installed](/docs/1.8/usage/cli/install/)
+
+Let's start with a simple example: an service that prints `Hello Marathon` to stdout and then sleeps for 5 sec, in an endless loop.
+Use the following JSON application definition to describe the application. Create a file with the name of your choice. 
+
+```json
+{
+    "id": "basic-0", 
+    "cmd": "while [ true ] ; do echo 'Hello Marathon' ; sleep 5 ; done",
+    "cpus": 0.1,
+    "mem": 10.0,
+    "instances": 1
+}
+```
+
+`cmd` in the above example is the command that gets executed. Its value is wrapped by the underlying Mesos executor via `/bin/sh -c ${cmd}`.
+
+Then, add the service to DC/OS
+
+```bash
+dcos marathon app add <your-service-name>.json
+```
+
+When you define and launch a service, Marathon hands over execution to Mesos. Mesos creates a sandbox directory for each task. The sandbox directory is a directory on each agent node that acts as an execution environment and contains relevant log files. The `stderr` and `stdout` streams are also written to the sandbox directory.
+
+## Using Resources in Applications
+
+To run any non-trivial application, you typically depend on a collection of resources: files and/or archives of files. To manage resource allocation, Marathon has the concept of URIs. URIs use the Mesos fetcher to do the legwork in terms of downloading (and potentially) extracting resources.
+
+Before we dive into this topic, let's have a look at an example:
+
+```json
+{
+    "id": "basic-1", 
+    "cmd": "`chmod u+x cool-script.sh && ./cool-script.sh`",
+    "cpus": 0.1,
+    "mem": 10.0,
+    "instances": 1,
+    "uris": [
+        "https://example.com/app/cool-script.sh"
+    ]
+}
+```
+
+What the example above does: before executing the `cmd`, download the resource `https://example.com/app/cool-script.sh` (via Mesos) and make it available in the service instance's sandbox. Verify that it had been downloaded by visiting the DC/OS web interface and clicking on an instance of `basic-1`, then on the **Files** tab. You should find `cool-script.sh` there.
```

Comment:
This sentence should use active voice (e.g. `The example above executes the `cmd`...`